client/src/modules/assets/modals/search.modal.html

```diff
@@ -23,40 +24,54 @@
             <bh-clear on-clear="$ctrl.clear('depot_uuid')"></bh-clear>
           </bh-depot-select>
 
-          <!-- Inventaire -->
+          <!-- Inventory -->
           <bh-inventory-select
             inventory-uuid="$ctrl.searchQueries.inventory_uuid"
             on-select-callback="$ctrl.onSelectInventory(inventory)"
             required="false">
             <bh-clear on-clear="$ctrl.clear('inventory_uuid')"></bh-clear>
           </bh-inventory-select>
 
+          <!-- inventory group -->
+          <bh-inventory-group-select
+            group-uuid="$ctrl.searchQueries.group_uuid"
+            name="group_uuid"
+            required="false"
+            on-select-callback="$ctrl.onSelectGroup(group)">
+            <bh-clear on-clear="$ctrl.clear('group_uuid')"></bh-clear>
+          </bh-inventory-group-select>
+
           <!-- lot number  -->
           <div class="form-group">
-            <label class="control-label" translate>STOCK.LOT</label>
+            <label class="control-label" translate>STOCK.LOT_ID</label>
             <bh-clear on-clear="$ctrl.clear('label')"></bh-clear>
             <input type="text" class="form-control" name="label" ng-model="$ctrl.searchQueries.label">
             <div class="help-block" ng-messages="ModalForm.label.$error" ng-show="ModalForm.$submitted">
               <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
             </div>
           </div>
 
-          <!-- entity -->
-          <bh-entity-select
-            entity-uuid="$ctrl.searchQueries.entity_uuid"
-            on-select-callback="$ctrl.onSelectEntity(entity)"
-            required="true">
-            <bh-clear on-clear="$ctrl.clear('entity')"></bh-clear>
-          </bh-entity-select>
+          <!-- expiration date -->
```

Comment:
It looks like this is `entry_date` not expiration date.

Reply:
Fixed